p2p/transport/quicreuse/options.go

```diff
@@ -10,9 +12,12 @@ func DisableReuseport() Option {
 }
 
 // EnableMetrics enables Prometheus metrics collection.
```

Comment:
Let's add to the comment saying if nil is passed, it will use the default registerer